docs/esasky/esasky.rst

```diff
@@ -234,10 +234,30 @@ To execute the same command as above you write this:
 The parameters are interchangeable in the same way as in
 :meth:`~astroquery.esasky.ESASkyClass.query_region_catalogs`.
 
+Get the metadata of specific observations or sources
+--------------
+If you already know the observation ID's or source names of interest,
```

Comment:
Heading should have the same length as the text above, not sure why sphinx didn't complain in this case

```suggestion
Get the metadata of specific observations or sources
----------------------------------------------------

If you already know the observation ID's or source names of interest,
```